test/cppsuite/tests/run.cxx

```diff
@@ -37,6 +37,14 @@
 #include "example_test.cxx"
 #include "hs_cleanup.cxx"
 
+namespace test_harness {
+/*
+ * FIXME-WT-7782: This needs to exist in a single translation unit. At the moment, almost everything
```

Comment:
I though we try to avoid mention of ticket IDs in comments?

Reply:
We generally do but it is acceptable, @tetsuo-cpp please leave a comment on WT-7782 as well indicating that this should be fixed in that ticket.

Reply:
In `FIXME` comments it is pretty normal. You can `grep` the codebase for `FIXME-WT` and see. I think it's a good practice because it means that when you close a ticket, you can search for `FIXME` comments related to your ticket and either remove them or address them. We started doing this because we had heaps of stale `FIXME` comments in the code that we forgot to remove.

Reply:
Sounds good, thanks for the explanation!